AsyncDisplayKit/ASTableView.mm

```diff
@@ -888,6 +888,18 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertSectionsAt
     return; // if the asyncDataSource has become invalid while we are processing, ignore this request to avoid crashes
   }
 
+  if (_contentOffsetAdjustmentTopVisibleRow != nil) {
+    __block NSInteger adjustedSection = _contentOffsetAdjustmentTopVisibleRow.section;
+    [indexSet enumerateIndexesUsingBlock:^(NSUInteger idx, BOOL *stop) {
+        if (adjustedSection <= idx) {
+          adjustedSection++;
```

Comment:
@zintus Should this be `if (idx <= adjustedSection)`? Aren't we interested in sections that were inserted above the top visible row? Also as a bonus, would you mind adding `else { *stop = YES; }`?